,target,doc
0,1,Import Error
1,1,**Describe the bug(__must be provided__)**
2,1,"I deploy Nebula on a cluster of 3 machines(Centos7), when I import data from csv by using https://github.com/vesoft-inc/nebula-importer. I met some error info, but I can't find any solution to solve it. "
3,,
4,1,**Your Environments(__must be provided__)**
5,1,  * OS: Centos7.6
6,1,  * Compliler: gcc version 4.8.5
7,1,  * CPU:
8,,
9,,1**How To Reproduce(__must be provided__)**
10,1,Steps to reproduce the behavior:
11,1,  1. deploy Nebula cluster
12,1,  2. define config
13,1,  3. run command to import data
14,,
15,1,**Expected behavior**
16,1,Import csv files successfully
17,,
18,1,**Additional context**
19,1,My config file:
20,0,[config.txt](https://github.com/vesoft-inc/nebula/files/3878623/config.txt)
21,1,My command:
22,0,go run importer.go --config ../example/config.txt
23,1,My error:
24,0,![image](https://user-images.githubusercontent.com/30515579/69410766-a5dfc980-0d46-11ea-836f-b6f97aa9d113.png)
25,,
26,,
